modules/n1ql/pages/n1ql-language-reference/dropsequence.adoc

```diff
@@ -0,0 +1,122 @@
+= DROP SEQUENCE
+:page-topic-type: reference
+:page-status: Couchbase Server 7.6
+:imagesdir: ../../assets/images
+:description: The DROP SEQUENCE statement enables you to drop a sequence in a given keyspace.
```

Comment:
In a scope.  SEQUENCES are SCOPE elements.

Reply:
Done